### ISSUE TITLE
FEATURE REQUEST: SNES CONTROLLER SKIN

### ISSUE THREAD

**Issue by Jeetkunndo** [GamePadViewer/feature-requests#3]
PLease add a snes skin to the options
![Proofshoot](https://github.com/GamepadViewer/feature-requests/assets/113386672/ba104c0c-453c-46eb-a783-8962dabde69c)

https://gamepadviewer.com/?p=1&css=https%3A%2F%2Fraw.githubusercontent.com%2Feseca%2Fcustom-gamepadviewer-skins%2Fmaster%2Fsnes%2Fibuffalo-map-skin.css&map=%7B%22mapping%22%3A%5B%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%223%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%222%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%222%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%223%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%221%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%220%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%220%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%221%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%229%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%229%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%228%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%228%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%225%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%227%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%224%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%224%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%2215%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%2215%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%2214%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%2214%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%2213%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%2213%22%7D%2C%7B%22targetType%22%3A%22buttons%22%2C%22target%22%3A%2212%22%2C%22disabled%22%3Afalse%2C%22choiceType%22%3A%22buttons%22%2C%22choice%22%3A%2212%22%7D%5D%7D

**Comment by unofficialdxnny** [comment]
if this one is made by yourself id recommend remapping it as some of the buttons arent mapped correctly

**Comment by Jeetkunndo** [comment]
this is not made by me, someone else did it, I AM ASKING that  should  include a SNES controller skin to the "default options" like the other skins like the xbox or the nes or the PS2/PS3/PS4 etc

**Comment by unofficialdxnny** [comment]
> this is not made by me, someone else did it, I AM ASKING that should include a SNES controller skin to the "default options" like the other skins like the xbox or the nes or the PS2/PS3/PS4 etc

NO! Gamepad Viewer is not meant to be a go to for skins as it is not a skin gallery... Thats what the community is for. 

This is what McPowned said "As much as I appreciate the enthusiasm for a new skin to be added, my priorities right now with gamepad viewer are focused on improving the the product overall, and less on adding one more skin
Were you unable to find info on how to create your own custom skin?"

**Comment by Jeetkunndo** [comment]
Oh im sorry i didnt mean in an offensive way.  im sorry if you missanderstood me, im not demanding. english is not my main. only was tryng to help, i tried to do it manually several times but doesnt work well most of the times. and that why i wondered if where possible, very sory didnt meat to insult you or your project. is an amazing project. hope you can still grow it more and have succed

**Comment by unofficialdxnny** [comment]
> Oh im sorry i didnt mean in an offensive way. im sorry if you missanderstood me, im not demanding. english is not my main. only was tryng to help, i tried to do it manually several times but doesnt work well most of the times. and that why i wondered if where possible, very sory didnt meat to insult you or your project. is an amazing project. hope you can still grow it more and have succed

no i didnt find you offensive :). I was just trying to help you so you dont keep waiting for it to be added. Its unlikely that new controllers will be added. Just possibly if a new console came out with a unique controller than maybe it will be added.
